include/sdf/Collision.hh

```diff
@@ -186,6 +186,24 @@ namespace sdf
 
     /// \brief Private data pointer.
     private: CollisionPrivate *dataPtr = nullptr;
+
+    /// \brief Set whether the lidar reflective intensity
+    /// has been specified.
+    /// \param[in] _laserRetro True if the lidar reflective intensity
+    /// has been set in the sdf.
+    public: void SetHasLaserRetro(bool _laserRetro);
```

Comment:
We haven't been consistently adding `Has` functions for optional elements. I only see this in `Camera.hh`. I'm not sure if that's the direction we want to go in, or if we want to just leave the default value in case it's unset.

Thoughts, @azeey ?